code/_onclick/hud/new_player.dm

```diff
@@ -1,35 +1,62 @@
+#define SHUTTER_MOVEMENT_DURATION 0.4 SECONDS
+#define SHUTTER_WAIT_DURATION 0.2 SECONDS
+
 /datum/hud/new_player
+	///Whether the menu is currently on the client's screen or not
+	var/menu_hud_status = TRUE
 
 /datum/hud/new_player/New(mob/owner)
 	..()
 
 	if(!owner || !owner.client)
 		return
 
-	if (owner.client.interviewee)
+	if(owner.client.interviewee)
 		return
 
 	var/list/buttons = subtypesof(/atom/movable/screen/lobby)
 	for(var/button_type in buttons)
-		var/atom/movable/screen/lobby/lobbyscreen = new button_type()
+		var/atom/movable/screen/lobby/lobbyscreen = new button_type(our_hud = src)
```

Comment:
one thing. does all this work before SSatoms initializes (right at the start of setup, players can join then and all)

Reply:
think it does, though I'd test it a wee bit to be sure; worst case, the player can reset the hud after setup

Reply:
I'd like it tested

Reply:
(it was tested and works fine, with the addition of `sleep` instead of `addtimer` the button now also properly re-enables after collapsing the menu during game setup)